### PR TITLE
Player 3498

### PR DESCRIPTION
**Pull request by pilievOoyala**

### CODE REVIEW

**Comment by aldigun** [comment]
LGTM.

File: test/unit-tests/ima_test.js [hunk 1]
```diff
@@ -80,8 +80,12 @@ describe('ad_manager_ima', function()
     };
     var content =
     {
-      all_ads : [ad]
+      all_ads : [ad],
```
Comment (aeng7) [inline test/unit-tests/ima_test.js]: Minor thing, extra comma

File: test/unit-tests/ima_test.js [hunk 2]
```diff
@@ -516,7 +521,52 @@ describe('ad_manager_ima', function()
...
+        notified = true;
+      };
+      initAndPlay(true, vci);
+      ima.playAd(amc.timeline[0]);
+      var am = google.ima.adManagerInstance;
+      am.publishEvent(google.ima.AdEvent.Type.STARTED);
+      expect(notified).to.be(true);
+    });
+
+    it('Play ad: Requests the AMC to hide the player UI', function()
```
Comment (aeng7) [inline test/unit-tests/ima_test.js]: Can we differentiate this description from the previous unit test? They currently have the same description

Comment (pilievOoyala) [reply test/unit-tests/ima_test.js]: Will do. Copy-pasting is bad :(

**Comment by aeng7** [comment]
LGTM

**Pull request merged**